docs/deployment/local-codebase.md

```diff
@@ -8,7 +8,6 @@ Update the `vagrant_synced_folders` configuration to sync your local Drupal code
 vagrant_synced_folders:
   - local_path: ~/Sites/my-drupal-site
     destination: /var/www/my-drupal-site
-    id: drupal
```

Comment:
Technically it's not necessary, it's only required if you want to do some special/custom things with the shares you create and not base them on paths.